client/elements/emoji-selector.html

```diff
@@ -1,3 +1,5 @@
+<base href="https://polygit2.appspot.com/emojilib+muan+1.0.1/components/">
```

Comment:
If we are going to use polygit for everything, we should probably just put the base in the HTML document is that it doesn't need to be hard-coded into the component.